.eslintrc.json

```diff
@@ -0,0 +1,117 @@
+        "keyword-spacing": "error",
+        "lines-between-class-members": "error",
+        "max-len": ["error", 120],
+        "max-params": ["warn", 4],
+        "max-statements-per-line": ["warn", { "max": 2 }],
+        "new-cap": "error",
+        "new-parens": "error",
+        "no-alert": "error",
+        "no-array-constructor": "error",
+        "no-caller": "error",
+        "no-confusing-arrow": ["warn", { "allowParens": true }],
+        "no-constructor-return": "error",
+        "no-dupe-else-if": "error",
+        "no-else-return": "warn",
+        "no-empty-function": "error",
+        "no-eq-null": "warn",
+        "no-eval": "error",
+        "no-floating-decimal": "warn",
+        "no-implicit-coercion": "warn",
+        "no-implied-eval": "warn",
+        "no-invalid-this": "error",
+        "no-iterator": "error",
+        "no-labels": "error",
+        "no-lone-blocks": "error",
+        "no-lonely-if": "error",
+        "no-loop-func": "error",
+        "no-mixed-operators": "error",
+        "no-multi-assign": "error",
+        "no-multi-spaces": "warn",
+        "no-multi-str": "warn",
+        "no-nested-ternary": "error",
+        "no-new-func": "error",
+        "no-new-object": "error",
+        "no-new": "warn",
+        "no-octal-escape": "error",
+        "no-octal": "error",
+        "no-param-reassign": "warn",
+        "no-proto": "error",
+        "no-redeclare": "error",
+        "no-return-assign": "error",
+        "no-return-await": "error",
+        "no-script-url": "warn",
+        "no-self-compare": "error",
+        "no-sequences": "error",
+        "no-setter-return": "error",
+        "no-shadow": "warn",
+        "no-template-curly-in-string": "warn",
+        "no-throw-literal": "warn",
+        "no-trailing-spaces": ["error", { "skipBlankLines": true, "ignoreComments": true }],
+        "no-undef-init": "error",
+        "no-unmodified-loop-condition": "warn",
+        "no-unneeded-ternary": "error",
+        "no-unused-expressions": "warn",
+        "no-useless-call": "error",
+        "no-useless-concat": "error",
+        "no-useless-constructor": "warn",
+        "no-useless-return": "error",
+        "no-var": "error",
+        "no-void": "error",
+        "no-whitespace-before-property": "error",
+        "object-curly-newline": ["error", { "multiline": true, "minProperties": 0 }],
+        "object-curly-spacing": "error",
+        "one-var-declaration-per-line": "error",
+        "operator-linebreak": ["error", "before"],
+        "padded-blocks": ["error", "never"],
+        "prefer-const": "warn",
+        "prefer-exponentiation-operator": "warn",
+        "prefer-regex-literals": "warn",
+        "prefer-rest-params": "error",
+        "prefer-template": "warn",
+        "quote-props": ["error", "always"],
+        "quotes": ["error", "double", { "avoidEscape": true }],
+        "require-await": "warn",
+        "rest-spread-spacing": "error",
+        "semi-spacing": "error",
```

Comment:
the behavior defined here may be confusing to someone reading the file, maybe write the definition as `["error", "last"]` instead

Reply:
I don't see `"last"` as a valid argument for [this option](https://eslint.org/docs/rules/semi-spacing).